src/sysdolphin/baselib/class.c

```diff
@@ -403,7 +428,7 @@ bool hsdObjIsDescendantOf(HSD_Obj* o, HSD_ClassInfo* p)
 
 void class_set_flags(HSD_ClassInfo* class_info, s32 set, s32 reset)
 {
-    class_info->head.flags = class_info->head.flags & ~reset | set;
+    class_info->head.flags = (class_info->head.flags & ~reset) | set;
```

Comment:
```suggestion
    class_info->head.flags = class_info->head.flags & ~reset | set;
```
Redundant

Reply:
Redundant, but fixes a clang warning, and is potentially helpful if you don't recall the bitwise operation precedence, so we're gonna keep it